samples/boards/st/t1s/README.rst

```diff
@@ -0,0 +1,153 @@
+.. zephyr:code-sample:: stm32_t1s
+   :name: 10BASE-T1S tests (OA_TC6)
+
+   Setup tests to examine long term stability of 10BASE-T1S communication
```

Comment:
Are we sure this is the correct location then ?
This has only little to do with STM.

What about moving this under net ?

Reply:
I was not sure where to place it..... Just wanted to make it looking correct with `make html-fast`.
I can move it to `net` - no problem.